ISISScientificComputing/autoreduce
# Instrument not found issue and views.py code cleanup

### Summary of work
* Fixes #762 by making the [instrument_summary](https://github.com/ISISScientificComputing/autoreduce/blob/762-instrument-not-found-when-instrument-exists/WebApp/autoreduce_webapp/reduction_viewer/views.py#L302) page output a more contextual error message "No runs found for instrument.",  "Instrument not found." and "An unexpected error has occurred when loading the instrument.". More could be added in the future if needed.
* Slight code cleanup of [reduction_viewer/views.py](https://github.com/ISISScientificComputing/autoreduce/blob/762-instrument-not-found-when-instrument-exists/WebApp/autoreduce_webapp/reduction_viewer/views.py)

![image](https://user-images.githubusercontent.com/5902731/93998755-e2c48480-fd8c-11ea-8989-f11e6e079ff3.png)

![image](https://user-images.githubusercontent.com/5902731/93998798-f4a62780-fd8c-11ea-94b2-6784cb1bbca4.png)


### How to test your work
* Code review 
* All unit tests pass
* Load an unknown instrument on the web app, e.g. `/instrument/ABCDE`. It should output "Instrument not found."
* Load an instrument with no runs. It should output "No runs found for instrument."

### Additional comments
* [views.run_list](https://github.com/ISISScientificComputing/autoreduce/blob/master/WebApp/autoreduce_webapp/reduction_viewer/views.py#L219) was removed in this PR as it appears to be redundant code. Please can someone else double check this.
* Removed references to [instrument.is_active](https://github.com/ISISScientificComputing/autoreduce/blob/master/WebApp/autoreduce_webapp/templates/instrument_summary.html) on the instrument_summary page as they were commented out. Please can someone confirm that this is redundant code.

Fixes #762
